src/main/pages/portable-workspaces/factories-getting-started.adoc

```diff
@@ -8,62 +8,69 @@ folder: portable-workspaces
 ---
 
 
-A Factory is a template containing configuration to automate generating new Workspace with factory identifier added to IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
+A factory is a template containing configuration to automate generating new Workspace with factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
 
 [id="try-a-factory"]
-== Try a Factory
+== Trying a factory
 
-Clone a public workspace on `che.openshift.io` by clicking the button below. https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]]
+To clone a public workspace on `che.openshift.io`, click the button: https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]].
 
 [id="using-factories"]
-== Using Factories
+== Using factories
 
-Factories can be invoked from a Factory URL built in multiple ways. You can replace the {% if site.product_mini_cli=="codenvy" %}`che.openshift.io` domain with the hostname of any Codenvy on-premises installation.{% else %}`localhost:8080` domain with the hostname of any Che installation.{% endif %}
+Factories can be invoked from a factory URL built in multiple ways. You can replace the {% if site.product_mini_cli=="codenvy" %}`che.openshift.io` domain with the hostname of any Codenvy on-premises installation.{% else %}`localhost:8080` domain with the hostname of any Che installation.{% endif %}
```

Comment:
The URL and name of what's being documented change here, but not the wording used. Considering the URL also changes in other places within this document (while leaving other aspects untouched), would it make sense to instead perform a conditional at the beginning of the document which assigns some attributes? It would be less maintenance-heavy to do so. :)

Example:

```adoc
{% if site.product_mini_cli=="codenvy" %}
:url: che.openshift.io
:install-type: Codenvy on-premises
{% else %}
:url: localhost:8080
:install-type: Che
{% endif %}
```

From that point on, you could use the attributes instead of using a lot of conditionals -- at least in most cases. :)